Submitted By: Robert Connolly
Date: 2004-12-08
Initial Package Version: 2.6.9
Origin: Jim Gifford - iproute2-2.6.9_041019-find_update-1.patch
Upstream Status: Send
Description: Fixes issues with Find version 4.2.5 and up

diff -Naur iproute2-2.6.9.orig/Makefile iproute2-2.6.9/Makefile
--- iproute2-2.6.9.orig/Makefile	2004-08-31 18:32:14.000000000 +0000
+++ iproute2-2.6.9/Makefile	2004-12-09 04:25:26.000000000 +0000
@@ -41,13 +41,13 @@
 	install -m 0755 -d $(DESTDIR)$(CONFDIR)
 	install -m 0755 -d $(DESTDIR)$(DOCDIR)/examples
 	install -m 0755 -d $(DESTDIR)$(DOCDIR)/examples/diffserv
-	install -m 0644 README.iproute2+tc $(shell find examples -type f -maxdepth 1) \
+	install -m 0644 README.iproute2+tc $(shell find examples -maxdepth 1 -type f) \
 		$(DESTDIR)$(DOCDIR)/examples
-	install -m 0644 $(shell find examples/diffserv -type f -maxdepth 1) \
+	install -m 0644 $(shell find examples/diffserv -maxdepth 1 -type f) \
 		$(DESTDIR)$(DOCDIR)/examples/diffserv
 	@for i in $(SUBDIRS) doc; do $(MAKE) -C $$i install; done
-	install -m 0644 $(shell find etc/iproute2 -type f -maxdepth 1) $(DESTDIR)$(CONFDIR)
-	install -m 0644 $(shell find man/man8 -type f -maxdepth 1) $(DESTDIR)$(MANDIR)/man8
+	install -m 0644 $(shell find etc/iproute2 -maxdepth 1 -type f) $(DESTDIR)$(CONFDIR)
+	install -m 0644 $(shell find man/man8 -maxdepth 1 -type f) $(DESTDIR)$(MANDIR)/man8
 	ln -sf $(DESTDIR)$(MANDIR)/man8/tc-pbfifo.8  $(DESTDIR)$(MANDIR)/man8/tc-bfifo.8
 	ln -sf $(DESTDIR)$(MANDIR)/man8/tc-pbfifo.8  $(DESTDIR)$(MANDIR)/man8/tc-pfifo.8
 
